src/WikiService.Validation/File/RemoveFileRequestValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests.File;
+using LT.DigitalOffice.WikiService.Validation.File.Interfaces;
+
+namespace LT.DigitalOffice.WikiService.Validation.File
```

Comment:
don't need validation for this endpoint;
user chooses concrete files; and if somehow there would be non-existing Id in list, repo just wouldn't delete it